src/components/LoadingPanel.js

```diff
@@ -0,0 +1,20 @@
+import { __ } from "@wordpress/i18n";
+import movingfiles from "../icons/movingfiles.svg";
+import { Spinner } from "@newfold/ui-component-library";
+
+
+export function LoadingPanel({ pluginName }) {
+  return <div className="nfd-flex nfd-flex-col nfd-items-center nfd-gap-5">
+    <h1 className="nfd-text-2xl">{__("Hold on while we get things setup for you!", "wp-module-ecommerce")}</h1>
+    <img src={movingfiles} className="nfd-w-50 nfd-h-50 nfd-text-[--nfd-ecommerce-text-dark]" />
+    <p className="nfd-text-base">
+      {
+        pluginName === 'ecomdash' ? 
+        __("Activating ", "wp-module-ecommerce")+`${pluginName}`+__(" plugin...", "wp-module-ecommerce")
+        : 
+        __("Activating the YITH ", "wp-module-ecommerce")+`${pluginName}`+__("plugin...", "wp-module-ecommerce")
```

Comment:
sprintf( __( Activating the YITH "%1$s" plugin..., "wp-module-ecommerce" ),             pluginName ), you can use this way that way, you don't have to use string concating

Reply:
will try to follow in next sprints

Reply:
Need to avoid this format as it throws a `warning` when we update pot file. 
<img width="1554" alt="Screenshot 2024-05-29 at 4 48 12 PM" src="https://github.com/newfold-labs/wp-module-ecommerce/assets/41899077/65cb6772-bee9-4b53-8ad8-d273512d9d6c">